scripts/generate_implemented_properties.js

```diff
@@ -9,8 +9,15 @@ const camelToDashed = require('../lib/parsers').camelToDashed;
 const dashedProperties = fs
   .readdirSync(path.resolve(__dirname, '../lib/properties'))
   .filter(propertyFile => propertyFile.substr(-3) === '.js')
-  .map(propertyFile => camelToDashed(propertyFile.replace('.js', '')));
-
+  .map(propertyFile => camelToDashed(propertyFile.replace('.js', '')))
+  .map(property => {
+    const isVendorSpecific = /^(o|moz|ms|webkit)-/.test(property);
```

Comment:
Sneaking in support for other vendor prefixes. I'm not sure if you're opinionated about other vendor prefixes. I'm looking at this purely from a testing perspective where I'd like to see support for all vendor prefixes. Otherwise it's hard to write tests that cover all browsers at once.

Reply:
The real bug is in: <https://github.com/jsdom/cssstyle/blob/62b3b1bfbeb6cdebf829de9caf777635e68c605d/lib/parsers.js#L712>

It should be:
```js
var first_segment = /^([^-]+)-/;
```